src/slic3r/GUI/GUI_App.cpp

```diff
@@ -2045,6 +2048,20 @@ void GUI_App::init_download_path()
     }
 }
 
+#if wxUSE_WEBVIEW_EDGE
+void GUI_App::init_webview_runtime()
+{
+    // Check WebView Runtime
+    if (!WebView::CheckWebViewRuntime()) {
+        int nRet = wxMessageBox(_L("Couldn't find Edge WebView2 Runtime.\nDo you want to install it?"),
```

Comment:
Would it be better to change the message to:
"Orca Slicer requires Microsoft WebView2 Runtime. Click Yes to automatically install it."
?